specification/quantum/resource-manager/Microsoft.Quantum/preview/2019-11-04-preview/examples/providerList.json

```diff
@@ -0,0 +1,87 @@
+{
+  "parameters": {
+    "subscriptionId": "1C4B2828-7D49-494F-933D-061373BE28C2",
+    "api-version": "2019-11-04-preview",
+    "locationName": "westus2"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "id": "QIO",
+            "name": "Quantum Inspired Optimization",
```

Comment:
also "type" is required

Reply:
Again, not ARM resources so type doesn't really apply.

Reply:
you are exposing these resources via an ARM api - so they automatically becomes ARM resources and must follow the arm rpc

Reply:
+1

---
In reply to: [546982699](https://github.com/Azure/azure-rest-api-specs/pull/11266#discussion_r546982699) [](ancestors = 546982699)

Reply:
not a typical ARM resource, see above comment. its a metadata kind of api and cant be created by end customer.